engineering/methodology.md

```diff
@@ -1,122 +1,90 @@
+after the Product Team and CTO will review this at the Product Sync meeting that
+happens once per week.
 
-The Kanban columns are defined as follows:
-
-### [Company Kanban](https://github.com/orgs/src-d/projects/9)
-
-Column      | Description
-------------|------------
-Backlog     | Only contains Epics and user stories.
-TODO        |
-In Progress |
-Done        |
-
-### Team Kanban
-
-Each team uses separate [Github Projects](https://github.com/orgs/src-d/projects/) as a Kanban board.
-
-Teams managing issues across different Github Orgs (i.e Applications, Language Analysis) are using the “add card” trick, described in [dear-github/dear-github#209](https://github.com/dear-github/dear-github/issues/209#issuecomment-357692288) to add issues from a different org.
-
-
-Column           | Description
------------------|------------
-Backlog          | Break-down of the Epics for each product on actual tasks
-TODO             | Tasks from backlog that are immediately ready to be acted on. Grooming a backlog to update this column is expected to be done on a daily basis by the Lead. See the Definition of TODO section below.
-Development      | Assigned task from TODO that time has been spent up on.
-Code Review (PR) | Issues that have PRs and are ready for Code Review.
-Deploy/Release   | Things that are implemented/merged, but have not yet been “delivered” to the users. This can be seen as a next release “backlog”. Release itself, although is not trivial and is documented <elsewhere>, is not expected to be handed in a separate issue.
-Done             | A task is done, when  a user can use the results of the work though established interface: package manager for libs, online for web services, etc. See the section below [What does TODO mean?](#what-does-todo-mean).
-
-On *"blockers"* - a task can be blocked at any stage of the development, so
-there is no separate column and Tag is used instead to signify if something is
-blocked.
-
-More details in [Definition](#definition) section below.
-
-### How to pick next issues to work on?
-
-Each team decides what works best for them. One effective way is picking tasks
-from **right to left** in the Kanban board, choosing from those issues that are
-neither assigned to someone nor blocked. A single person should not be in charge
-of delivering the whole issue from Development to Done. In that way knowledge is
-better spread across the whole team.
-
-For this to work properly, issues must be documented so that a person can move on with the issue, reading what has been done.
+The [Backlog repository](https://github.com/src-d/backlog) contains the full
+backlog for the Engineering Team and Product Team; these issues are managed by
+the Head of Product and only him can create new issues there. Each issue at this repository represents an EPIC, and this EPIC should be linked to his Design
```

Comment:
```suggestion
the Head of Product and only he can create new issues there. Each issue in this repository represents an EPIC, and that EPIC should be linked to it's Design
```

Reply:
nit: …only they can…

(to remove gender specificity from the term)

Reply:
nit: …linked to its Design

(it's = it + is, its = it-possessive)